qa/common/infer_util.py

```diff
@@ -153,37 +162,83 @@ def infer_exact(tester, pf, tensor_shape, batch_size,
 
             # create and register shared memory region for inputs and outputs
             if shm_region_names is None:
-                shm_ip0_handle = shm.create_shared_memory_region("input0_data", "/input0", input0_byte_size)
-                shm_ip1_handle = shm.create_shared_memory_region("input1_data", "/input1", input0_byte_size)
-                if "OUTPUT0" in outputs:
-                    shm_op0_handle = shm.create_shared_memory_region("output0_data", "/output0", output0_byte_size)
-                if "OUTPUT1" in outputs:
-                    shm_op1_handle = shm.create_shared_memory_region("output1_data", "/output1", output1_byte_size)
+                if not _match_registered(registered_shm_regions, '/input0', input0_byte_size):
+                    shm_ip0_handle = shm.create_shared_memory_region("input0_data", "/input0", input0_byte_size)
+                    registered_shm_regions['/input0'] = (input0_byte_size, shm_ip0_handle)
+                else:
+                    shm_ip0_handle = registered_shm_regions['/input0'][1]
+
+                if not _match_registered(registered_shm_regions, '/input1', input0_byte_size):
```

Comment:
should this be using `input1_byte_size` instead?

Reply:
Yes. Although in non-variable shape tensors input0_byte_size = input1_byte_size. Fixed now.